src/main/java/io/github/dsheirer/record/wave/AudioMetadataUtils.java

```diff
@@ -89,38 +89,35 @@ public static Map<AudioMetadata, String> getMetadataMap(IdentifierCollection ide
 
         if(identifierCollection != null)
         {
-            for(Identifier to: identifierCollection.getIdentifiers(Role.TO))
-            {
-                StringBuilder sb = new StringBuilder();
-                sb.append(to.toString());
+            StringBuilder sb;
 
-                List<Alias> aliases = aliasList.getAliases(to);
+            Identifier to = identifierCollection.getToIdentifier();
+            sb = new StringBuilder();
+            sb.append(to.toString());
 
-                if(!aliases.isEmpty())
-                {
-                    sb.append("\"").append(Joiner.on("\",\"").join(aliases)).append("\"");
-                }
+            List<Alias> toAliases = aliasList.getAliases(to);
 
-                audioMetadata.put(AudioMetadata.TRACK_TITLE, sb.toString());
-                break;
+            if(!toAliases.isEmpty())
+            {
+                sb.append("\"").append(Joiner.on("\",\"").join(toAliases)).append("\"");
             }
 
-            for(Identifier from: identifierCollection.getIdentifiers(Role.FROM))
-            {
-                StringBuilder sb = new StringBuilder();
-                sb.append(from.toString());
+            audioMetadata.put(AudioMetadata.TRACK_TITLE, sb.toString());
 
-                List<Alias> aliases = aliasList.getAliases(from);
+            Identifier from = identifierCollection.getFromIdentifier();
+            sb = new StringBuilder();
+            sb.append(from.toString());
```

Comment:
There's potential for 'from' identifier to be null here.  Please add null check/safety.